WordPress/gutenberg
# Scripts: Update `puppeteer-core` dependency

## What?
Upgrades the `puppeteer-core` package to the latest version (23.1.0).

## Why?
This PR fixes the issue with `@wordpress/scripts` having five high-severity vulnerabilities introduced with an older version of the `puppeteer-core` package.

See: #63771

## How?
Removes the sub-dependencies versions with vulnerabilities.

## Testing Instructions
1. In a WP theme development environment, run `npm install @wordpress/scripts path webpack-remove-empty-scripts --save-dev`
2.  Run `npm audit` and you'll see a warning for 5 high-severity vulnerabilities.
3. Add the following to your package.json file: 
    ```json 
    "overrides": {
        "puppeteer-core": "^23.1.0"
    }
    ```
5. Run `npm install` to update the packages.
6. The vulnerabilities are now removed.
